CondFormats/BTauObjects/src/BTagEntry.cc

```diff
@@ -17,8 +22,8 @@ BTagEntry::Parameters::Parameters(
   float discr_max
 ):
   operatingPoint(op),
-  measurementType(measurement_type),
-  sysType(sys_type),
+  measurementType(makeLowerCase(measurement_type)),
+  sysType(makeLowerCase(sys_type)),
```

Comment:
@HeinAtCERN 
This is a bug: never return a reference to a local object! When `makeLowerCase` function completes, its storage is freed, and `data` goes out of scope. Hence, references to local objects refer to memory that is no longer valid. Fix it by returning by value: it is a copy, but the compiler should optimise.